### DISCUSSION TITLE
Versions Prior to 3.4.1

### DISCUSSION THREAD

**Discussion by kokorzeniowski** [Cog-Creators/Red-DiscordBot#5991] [Q&A] [answered]
https://cve.mitre.org/cgi-bin/cvename.cgi?name=2020-15278

Red Discord Bot before version 3.4.1 has an unauthorized privilege escalation exploit in the Mod module. This exploit allows Discord users with a high privilege level within the guild to bypass hierarchy checks when the application is in a specific condition that is beyond that user's control.

has this bug been patched and if so does anyone have the directory prior to the patch, let me know asap testing this bug for a project directing privilege escalation project for discord.

**Comment by Drapersniper** [comment]
On the link you linked yourself says "we still highly recommend updating to 3.4.1 to completely patch this issue."

**Comment by Jackenmen** [comment, marked as answer]
As explained in the CVE (which was requested by us), the feature is only present in versions prior to 3.4.1 and users can patch it by updating to version or 3.4.1 or higher (we recommend always keeping your version up-to-date which is 3.4.18 at the time of writing).
